rest-backend/pom.xml

```diff
@@ -26,8 +26,8 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>1.5.22.RELEASE</version>
-		<relativePath/>
+		<version>2.5.0</version>
```

Comment:
Why not 2.5.1?

Reply:
Because it was  not yet released when i worked on it and didn't noticed it was out :) i will update it